From 32243b2148fd0dacd0630affaea59345c64df79a Mon Sep 17 00:00:00 2001
From: Felix Fietkau <nbd@nbd.name>
Date: Tue, 2 Nov 2021 10:36:14 +0100
Subject: [PATCH 051/102] bpf-headers: unset PKG_CONFIG_PATH

This fixes an issue where the kernel would pick up an incompatible target
libyaml for building host tools

Signed-off-by: Felix Fietkau <nbd@nbd.name>
---
 package/kernel/bpf-headers/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/package/kernel/bpf-headers/Makefile b/package/kernel/bpf-headers/Makefile
index 5f5b89370d..df24bfa13f 100644
--- a/package/kernel/bpf-headers/Makefile
+++ b/package/kernel/bpf-headers/Makefile
@@ -41,6 +41,8 @@ define Package/bpf-headers
   HIDDEN:=1
 endef
 
+PKG_CONFIG_PATH:=
+
 export HOST_EXTRACFLAGS=-I$(STAGING_DIR_HOST)/include
 
 KERNEL_MAKE := \
-- 
2.25.1

